packages/atomic/src/components/commerce/product-template-components/atomic-product-price/e2e/atomic-product-price.e2e.ts

```diff
@@ -0,0 +1,93 @@
+import {test, expect} from './fixture';
+
```

Comment:
I think we should also test that the price is formatted according to the engine's configured `context.language`, `context.country`, and `context.currency`.